src/pages/project/[id].tsx

```diff
@@ -0,0 +1,55 @@
+import DashboardLayout from "@/layouts/DashboardLayout";
+import { ObjectId } from "mongodb";
+import { Project } from "@/types/project";
+import { GetServerSideProps } from "next";
+import { getSession } from "next-auth/react";
+import { findProjectById } from "@/helpers/project";
+
+const ProjectDetailsPage = ({
+  // id,
+  name,
+}: // participants,
+// tickets,
+// swimlanes
+Project) => {
+  return (
+    <DashboardLayout>
+      <h1> {name}</h1>
```

Comment:
Please put a space after {name}

Reply:
ya that's just a placeholder, we're going to remove that anyway. This is where the board component that you're working on will live @mhaeco